### PR TITLE
Games selector

### PR DESCRIPTION
**Pull request by mankinchi**
- Change language selector to more generic selector
- Add Games selector
- Switch game

### CODE REVIEW

**Comment by mankinchi** [comment]
Trying to solved https://github.com/ericadamski/alphabet-keys/issues/56

File: src/index.js [hunk 1]
```diff
@@ -2,11 +2,17 @@ import React, { Component } from 'react';
...
-import EMOJIS, { SUPPORTED_LANGS } from './data/emojis';
-import * as serviceWorker from './serviceWorker';
-import Selector from './language-selector'
+import EMOJIS from './data/emojis';
+import { SUPPORTED_LANGS } from './data/languages';
+import { Games } from './data/games';
+
+import Selector from './selector';
+
+console.log(SUPPORTED_LANGS);
```
Comment (ericadamski) [inline src/index.js]:
```suggestion
```

File: src/selector.js [hunk 2]
```diff
@@ -50,11 +42,17 @@ const SelectorListWrapper = styled.div`
...
   display: flex;
 
-  .is-open & {
-    pointer-events: auto;
-    opacity: 1;
-    transition: opacity 0.5s ease;
-  }
+  ${({ isOpen }) => {
+	if (isOpen) {
+	  return `
```
Comment (ericadamski) [inline src/selector.js]: this should use the `css` function from styled-components https://www.styled-components.com/docs/api#css

```suggestion
	  return css`
```

File: src/selector.js [hunk 3]
```diff
@@ -108,26 +110,24 @@ const Close = () => (
...
     });
   };
 
-  toggle = e => {
-    const { open } = this.state;
-    this.setState({
-      open: !open
-    });
+  toggle = () => {
+	console.log(this.state.open);
```
Comment (ericadamski) [inline src/selector.js]:
```suggestion
```

**Comment by ericadamski** [comment]
Check out your changes live https://alphabet-keys-git-fork-mankinchi-gamesselector.ericadamski.now.sh 💃 @mankinchi

File: src/selector.js (hunk 2 above)
Comment (mankinchi) [reply src/selector.js]: since this is just a string, I don't think we need to run it through the css function

**Comment by ericadamski** [comment]
@mankinchi can you also check that the instructions on adding a new language in the  README file still apply? Just want to make sure it is simple for others to add by keeping the instructions up to date

**Comment by mankinchi** [comment]
@ericadamski it's out-of-dated but I haveepudated it

**Comment by mankinchi** [comment]
@ericadamski was hoping to leave it till Hacktoberfest but it's fine :)). Give me more issues and I will handle them ;)

**Comment by ericadamski** [comment]
I think you have to open the PR in October for it to count

**Pull request merged**